prow/cmd/status-reconciler/main.go

```diff
@@ -73,6 +73,7 @@ func gatherOptions() options {
 	fs.StringVar(&o.configPath, "config-path", "/etc/config/config.yaml", "Path to config.yaml.")
 	fs.StringVar(&o.jobConfigPath, "job-config-path", "", "Path to prow job configs.")
 	fs.StringVar(&o.pluginConfig, "plugin-config", "/etc/plugins/plugins.yaml", "Path to plugin config file.")
+	fs.StringVar(&o.gcsCredentialsFile, "gcs-credentials-file", "", "File where Google Cloud authentication credentials are stored. Required for GCS writes.")
```

Comment:
Lol so this never worked? I think we should start naming the flag provider-agnostic, something like `blob-storage-credential-file` and deprecate the old `gcs-credentials-file`  flag

Reply:
I guess yes. But I also didn't find any example with the status-path flag in the repo :)

Then I'll also add the blob-storage-credentials-file flag in all affected binaries here

Reply:
Done

Reply:
I'll remove the old flag. As far as I can see there was no way to set it